### ISSUE TITLE
CSS Jitter at High Resolution

### ISSUE THREAD

**Issue by hectorsector** [github/training-kit#470]
## Overview
**TL;DR**
There is a jitter that occurs at high resolutions, see https://github.com/github/training-kit/pull/469#issuecomment-280109791 for complete details. 

### Extra Detail
https://github.com/github/training-kit/pull/469#issuecomment-280109791

### How to Contribute
- [ ] Comment on this issue if you'd like to work/collaborate on it! 
- [ ] Fork the [repository](https://github.com/github/training-kit).
- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://services.github.com/on-demand/intro-to-github/) is a refresher course if you're unsure about how to make a change on GitHub.
- [ ] Push your changes to your repository. 
- [ ] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/on-demand`.

### Questions?
- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!

**Comment by ebenezergraham** [comment]
Hey @hectorsector I would like to help. 

NB: This is my first contribution to an open source project and I would really appreciate it if we can collaborate to resolve the jiggle

**Comment by hectorsector** [comment]
👋 that's fantastic @ebenezergraham!

I haven't tested in quite some time, and probably won't be able to until I get back to my retina display on Monday. If you have a high-res display I would start by seeing if you can replicate it. If you can, then I'm happy to collaborate.

Also pinging @beardofedu who was the first one to notice this and may be able to test this before me.

**Comment by beardofedu** [comment]
Sorry for the late response @hectorsector / @ebenezergraham I can test the solution to the jiggle whenever.

**Comment by hollenberry** [comment]
Closing this out since our on demand courses are now deprecated.

Thank you for sharing this idea and starting this work!
